packages/utils/telemetry-utils/src/error.ts

```diff
@@ -0,0 +1,85 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	FluidErrorTypes,
+	IGenericError,
+	IErrorBase,
+	ITelemetryProperties,
+	IThrottlingWarning,
+	IUsageError,
+} from "@fluidframework/core-interfaces";
+
+import { LoggingError, wrapErrorAndLog } from "./errorLogging";
+import { IFluidErrorBase } from "./fluidErrorBase";
+import { ITelemetryLoggerExt } from "./telemetryTypes";
+
+/**
+ * Generic wrapper for an unrecognized/uncategorized error object
+ */
+export class GenericError extends LoggingError implements IGenericError, IFluidErrorBase {
+	readonly errorType = FluidErrorTypes.genericError;
+
+	/**
+	 * Create a new GenericError
+	 * @param message - Error message
+	 * @param error - inner error object
+	 * @param props - Telemetry props to include when the error is logged
+	 */
+	constructor(message: string, readonly error?: any, props?: ITelemetryProperties) {
+		// Don't try to log the inner error
+		super(message, props, new Set(["error"]));
+	}
+}
+
+/**
+ * Warning emitted when requests to storage are being throttled.
+ */
+export class ThrottlingWarning extends LoggingError implements IThrottlingWarning, IFluidErrorBase {
```

Comment:
This one is only used in container-loader, I think we could move it there instead and stop exporting it anywhere.

Reply:
Thanks! Will update!